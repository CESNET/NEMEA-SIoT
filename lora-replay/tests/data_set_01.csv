uint32 SIZE,uint64 TIMESTAMP,string PHY_PAYLOAD
16,1504704096,D6FF1158C47462DB
30,1504704124,048A5A012680EDADC8CF3B44619F62
22,1504704300,E138A87DC744CE9022A0NY
454,1504704334,E3889F6E65FEA7630544AF521404F745A03B7B87F35DF35C85E44A5E3CF5AC10EA56783183ABB14002BDB394A5647894C61B0B618A10005CC302B6FA959337DF12B77A83180FCE7A94D0B08EE5E2CBFC6CDF6DE0BB0E0D440BD3B22DEFD05E2C90C7E5BE7E22BD70DEB3B40F8A15363BE215FE20D3302305097CF6C1391E4175A19AD63A707EF4A4DBE5D13CFE81C78A12BEDE009FE8E6E765EDB68C2FF778837045ABE301F57A2552A5E0C5F1EC8AC976C20C4FEEFB31F89BBB477ACF6F73C510A5EBD6E8042FF4F6A252B04B8F50B72B2D2CB276BE19A26C252152539B184D7D9C99
68,1504704374,037DB8DD36EA0F8CF3353AFB74508AF6AAB93D3A2F4061B7EE15F362A0707BB9B9C2
22,1504704682,C76C368D61AE9FF5DA8A11
64,1504705135,0A0ADA664F5048D4EDC1EEB3D346E38DACD0E86FDFFE7A8C155806F2A88DE627
120,1504705577,0B28E53D0E7879FC137F3EADD119D18D94A7519CE77B1CF95BEA4E0660DDADA8119EAC953BA692D1E163928ADABE64B4681517A08B9EAF710E0B30eY
323,1504705940,C269BBF221CBD7ECD96350510E6888E98539F44B624295A28080465E986C0037C0FC4E7D71B5C190D9EF81BC383131961DA5A028AAA863EB2AD73A3495594876E0A9153125159B5E514A5C397D6C73C84C53BCE3F17763459C3DE5214E8AD24113302552A423C9ACF8421BB899A1E03271D83DD88C0FD404F94199CD52915627867B18B4A073A407C32F63CC7B03C1817BAAD456C2BC966ACB6EFD356495834E%DY
256,1504706116,E7D6AD166082F8F2261A47BBAD14D749219DCC2E7FD23F4367EA24D01135E1A1FC64EE0ABF3F43830FC187903C19FF0118F6AF1C540E924D58B122E1320AF1E15AE5AD9217BDC9B2E87C976F4654C637BF12BBAC4B90CCD81689B22A3738B8A897FD98C290FFDF3BD7B8ED2CBF46104C4F9C217F94FDA2B64F7FBF9E761111CE
30,1504706251,155C1001260067000190DDCDE864/Y
363,1504706351,D8975B84FD28774792C67E9FBFF2513656B80F192A1D524AFDF14153CA12D447089CB9361CEE0F03FFF8A4709CD9D668E3D26FEAB7DAE91AB2491A1CF5FAAD881C68D7EABCC6BBE6323180B55CBD7F5AE22A200716D07CD6ADFC3D4E5D6D91482D20BD93B4F43EE44969BD81BC44B1C15B18D505BB6FBCDE7B228AF26737E82A9FB8F6176D8A8BB8600E7FB74CB0A420C0A54510AFEFB52B3FF9760B6D90C068AB458545CADCAB5075C9318398380BD6F65E886CDDY
356,1504706471,DE4966672B1654F940936E392A69BC3229D1B5A4943FFF895A2182A704587E851E3DB6044FC13191274256D741A5BD6970ABD21614B9BE0AAF7B68039B1AA590B5680B059EBC5C00F99AD2AF1BD38C875BAD0D0A25F4165CF46FE130DD98E25B1C63E787566E1FF54ECC63EE4371CA12B490F276E0D7BF55229B2F98A271E185405EF50E64801E70CC8CCF0F3C928DF9828A63F73F323B6EEC01AD3ED639ED3A7F3200DC47F2E5BFA48DA25C9AD2BD1366C7
144,1504706907,D1077F3084C60E4B21036ED90D1974F9803F4914A78B62EC9B2BAAF12298AC0C2FEDE2455AAFCB12A867B9EBCD0D696AF8C0D04392B6925ACD2EEB072906DF32E7F0939CE3229BiY
246,1504706921,0D2AAEAC3CC3B5E230824B960E89308567695AAC8D313963A1923CD1DE22560E24AEA023243C7E6473277194DADAB87A08168513A93E9E3ECD952E0D7D7EE3983DCF049608C1709720833294787238182B624F25FD2F2E595A72FAC90669AE7A7A467B055B987371508C43AAAF62CA803243FB5F9773194E5981 Y
356,1504707082,C5F6D538FDCF47ADB1988C8449A22A9BBA839A560BC0B67F02DFBF8462756752056993E6D0645C297916DCA1E3A4A03383EDCB86C3716E1A96C714582A36038EEBB7804CA0AC4A2D9B9F729A5CEE44482D8F8DFE42472B1AE310D72FB77D7D9D58739A23810A4EC3CABEE38236A9F5187020809DAD882EFF9AE5D9C82E1C013B2A20D9A3EFE321F6ACC17F8E5EE5FDC00E2FCE181D54DB4BC535266A5619B53E27DCBA0D66BE0FA0359CD8C533F6F06E680E
102,1504707563,F4DEB89EA03F5AD8621A08FCBF3DCB035995D6F5C2D275564D2413A15221EA6AFE3992F17FF9D78CAC78CC3CFE11DE53EFC472
29,1504708306,E00910012600880001AE3D12C18DY
450,1504708537,DEFDEB78000B45F23FF90DFF9FC64143D5C7EF802CB8A8C63A0B8C7C658402DFBE39FB49A53454B23A657E9EFA6616064FD6E0DAD3383314C1D42D18BBEEA4D183AC0007DF896D38A570B390E004DF6A6245B3AD5FB2247A0C322103B60E06308B22C191FC7DD0BDE132CF6610545350C2EC8759A57ADB1A640CAEAEE4C3ED6BD81810BA116F81A3201954D53782A397DCF346276FAE043547CC90FC6D725282EE02551290D859A7AB209395D6C58A3F870E190B7FA8FA66C1E6F8A3C3523661B5809DF787D2C84843ABD926E089FE657E5504D13CDA8D05B860EC165E895EA96D
68,1504708722,DE31DF32E4B3E9A809257474D228A006E32066B4379C8595EECA4602158D2E7284CF
68,1504885602,40D12B0926001948018F512F67A264DD8535396CEDF9EC2FC00CAC1AED6CE2C96AE2
68,1504885466,40D12B092600174B015B45BEA7DC6A19D35C2C128B5F48A15EBF29017F844DA9FD7F
68,1504885602,40D12B0926001948018F512F67A264DD8535396CEDF9EC2FC00CAC1AED6CE2C96AE2
68,1504887369,40B12B0B26003347C117B7A3675621145926F4012EF2CC4034219CEA8E18420B70FB
68,1504952168,49D12BC126C02C4E91D87706F298E328431C213279FE5775B6F47B61322D1E011344
468,1504883337,7E25D25C28DDC0997EFAF1477C717B8FD5E70C316CDA034C85F8AA895B6F839D2E25F299F0AD5C3767C6F70A9C5C8C80D9C2AC02128B196DC3FCA8FC658AFA00B71D2EE5C7210AD8AAFB1A9203B79B33C48CF77F92A3BEFC9B1EE42788C53E489D52D1A62D319AFBD72956D9DA69D3FD3ECEFC734A66B85EC26F7F6609D650123F597270671EA517FF0804704B695258B4B371A8DAE993522A1AC3AFD2EC3A54C703A7ABEF5C904FC7B8EF29943B2F9A2E834E33592E17B4AF9DE5B249ACB04DB3CE32467CDFD7E845A7D6C2EA1776E73C96D70FE18B73F9E7DCCBEBD07C8BFD3E9261219C50CD1D8CoY
44,1504779330,40C1B14E0180010091922FDD3B38D165C2B12A0C6AD3
44,1504790080,40C1B14E0180020008D776C57C0456596E828FE46059
44,1504793663,40C1B14E01800300089A77F3AB8F519670B1D78DF5AB
44,1504797247,40C1B14E0180040008C5384C148F37572AD01A38574F
44,1504804413,40C1B14E018005000883752CEACF997DFC74CB38AFCB
44,1504815166,40C1B14E0180060008A41F5F97DFA384D70F438374B7
44,1504818752,40C1B14E0180070008503082AAF265E8CE460BD5F3CB
44,1504822338,40C1B14E01800800086257D47EC10D83D574A7F40BB6
44,1504833097,40C1B14E0180090008B72A13BE2063E3F41925FD1BD0
44,1504836682,40C1B14E018000000863B257977ED9A7DCD8C14AAA7E
44,1504840269,40C1B14E0180090008B72A13BE2063E3F41925FD1BD0
44,1504843855,40C1B14E01800A00088ED419FB19FB591A994FF4BE77
44,1504847442,40C1B14E01800B00083381343FEC67748C138EE421A0
44,1504851028,40C1B14E01800C00080431A32DCAD61236F3598C1F2E
44,1504854615,40C1B14E01800D00084A0492EBBF2DA1599DD606D970
44,1504858201,40C1B14E01800E0008DB38FF02E24B4FD2F8B726E5DC
44,1504861787,40C1B14E01800F0008208FA18953A348BBB2B3D71ECD
44,1504865372,40C1B14E01801000C03E85CD317615C764DD20A459E8
44,1504872543,40C1B14E0180110008004BCCD84E116C787AE1C2AE18
44,1504876128,40C1B14E0180120008A2751ACD530EC880D6D815AB5A
44,1504879712,40C1B14E01801300085262EB062E810876CE47C10F1E
44,1504883295,40C1B14E0180140008085D62CE81366E210D572F6746
44,1504886879,40C1B14E018015000836AEEAD59C5B47349803D3E8FB
44,1504883295,40C1B14E0180160008085D62CE81366E210D572F6746
44,1504886879,40C1B14E018017000836AEEAD59C5B47349803D3E8FB
44,1504890464,40C1B14E01801800089B5E55C2ADE1347E3BFB4466BB
44,1504901218,40C1B14E01801900087B731D79A0E2258D5846AA975D
44,1504904803,40C1B14E01801A000893ACEE4536F2E5A1C83BBF13BC
44,1504911976,40C1B14E01801B000899D0413BC99E342C6239A1B73E
44,1504915562,40C1B14E01801C0008913802C31ADDC694B21D7C9DDA
44,1504919148,40C1B14E01800000089D7B18351CC3FD16AC2CEC5E07
44,1504922733,40C1B14E01801C0008913802C31ADDC694B21D7C9DDA
44,1504926318,40C1B14E01801D0008D8C60BCDFE5C4A29FCDFE236BD
44,1504929903,40C1B14E01801E0008DC57FE3541091ECC8E092E58DC
44,1504937078,40C1B14E01801F00083A1EF50C01AA130ECB198BDC59
44,1504940666,40C1B14E01802000083C67FC7A7459833F62539DD1A8
44,1504944255,40C1B14E0180210008A297AFBAB3E74B2FE161FECF8C
44,1504947844,40C1B14E018022000899F02D3B3F86EF28F600916422
28,1504960051,40091001260051100104CD270F05
28,1504960113,4009100126005210014367900F25
28,1504960238,400910012600531001C8F865C8EE
28,1504960300,400910012600541001C1A74CF727
28,1504960362,400910012600551001BC86657464
28,1504960487,400910012600561001F8446CB288
28,1504960549,400910012600000001383CF1975E
28,1504960611,400910012600561001F8446CB288
28,1504960674,4009100126005710014EEBC20A1D
28,1504960736,400910012600581001EF26F53465
28,1504960798,400910012600591001436C69C04B
28,1504960923,4009100126005A1001E91E7B9AC6
28,1504961172,4009100126005B10010A093DED28
28,1504961297,4009100126005C1001D65B1D24D1
28,1504961359,4009100126005D1001B07F1BAB2D
28,1504961421,4009100126005E1001730CA450B6
28,1504961484,4009100126005F1001025080C083
28,1504961546,400910012600601001D33977EF7A
28,1504961670,40091001260061100183F77AF69B
28,1504961733,400910012600621001E938A30958
28,1504961795,400910012600000001BA20BBED89
28,1504961857,40091001260001000100D10937D1
28,1504961920,400910012600020001209FE8CC8A
28,1504961982,40091001260003000148F6DAB004
28,1504962044,4009100126000400010446044C01
28,1504962106,400910012600050001A0A1AB74D5
28,1504962169,40091001260006000164FA37F511
28,1504962231,4009100126000700012F3156EBFD
28,1504962293,400910012600080001E4E1ACEDA1
28,1504962418,4009100126000900019766B3FDBB
28,1504962480,4009100126000A0001B04B6EAE48
28,1504962543,4009100126000B00010FC08374A9
28,1504962605,4009100126000C000141C64243EF
28,1504962667,4009100126000D0001CDDF62D213
28,1504962729,4009100126000E00018E993DD60F
