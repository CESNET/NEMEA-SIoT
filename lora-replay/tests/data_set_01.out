76099889,2017-09-08T05:11:09.000,0,"Replay attack has been detected for the device 76099889","0009"
26099889,2017-09-09T04:05:33.000,0,"Replay attack has been detected for the device 26099889","001C"
66099889,2017-09-09T14:36:51.000,0,"Replay attack has been detected for the device 66099889","1056"
